modules/ossm-deploy-single-mesh.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+[id="ossm-deploy-single-mesh_{context}"]
+= Single mesh deployment model
```

Comment:
I'm not entirely sure what to do with this one. It's sort of overlapping with single tenant-- "single mesh" isn't really a deployment model other than "you have a mesh but you didn't federate it with another"

Reply:
Until we roll out Federation it's the only option customers currently have.  And customers don't have to federate their meshes if they don't want to.